Zunächst wird eine neue Klasse MyException definiert. 
In dem \lstinline$try-Block$ wird nach der Konsolen-Eingabe mit der \lstinline$raise$-Anweisung eine Ausnahme ausgelöst, 
wobei der vom Benutzer eingegebene Wert als Argument übergeben wird.
\lstinputlisting[language=Python, firstline=1,lastline=25]{exercises/src/exceptionhandling03.py}
